owsi-core/owsi-core-components/owsi-core-component-wicket-more/src/main/java/fr/openwide/core/wicket/more/notification/service/AbstractNotificationContentDescriptorFactory.java

```diff
@@ -38,6 +64,20 @@ public Component get() {
 					},
 					locale
 			);
+			
+			
+			IHtmlNotificationCssRegistry cssRegistry = null;
+			if (cssService.hasRegistry(getVariation())) {
+				try {
+					cssRegistry = cssService.getRegistry(getVariation());
+				} catch (ServiceException e) {
+					LOGGER.error("Unable to load the CSS file", e);
```

Comment:
Shouldn't we chain an exception here? The resulting email will be very badly formatted if we have no CSS file.